R/utilities.R

```diff
@@ -913,3 +913,31 @@ track_recent_covid_increases <- function(
   }
   return(out)
 } 
+
+update_fac_outbreaks_sheet <- function() {
+  scrape_df <- behindbarstools::read_scrape_data(all_dates = TRUE)
+  outbreaks_sheet_loc = "1I7oubSBZT1GnDL30f4jHzIQwQGso5RulrrBUgxFfRAM"
+  metrics <- c("Residents.Confirmed", "Residents.Active", 
+               "Residents.Deaths", "Staff.Confirmed", "Staff.Deaths")
+  out <- metrics %>%
+    map(~ track_recent_covid_increases(metric = .x, 
+                                       scrape_df = scrape_df))
```

Comment:
So to me, the quickest fix for that would be just to add `outbreaks_sheet_loc = outbreaks_sheet_loc` here. But also I don't think that `track_recent_covid_increases` needs to have a default value for the sheet location. (As best I can tell, we only use `track_recent_covid_increases` once). So I think it would be better to remove the default value in `track_recent_covid_increases` and then also add the  `outbreaks_sheet_loc = outbreaks_sheet_loc` to the function call here, if you have the time.

Reply:
Great suggestion, thanks! Similarly, I decided to remove the hard-coded argument for `delta_days` so that if we want to change the outbreaks spreadsheet to looking back over the past two weeks rather than one week, we can easily do so from `post_run.R`.